README.md

```diff
@@ -235,25 +257,16 @@ class Foo {
 new Foo().queueMethod();  // will log 1, rather than undefined
 ```
 
-One possible implementation, based on `@register`:
+The `@initialize` decorator could be used to ensure that, on construction of a class, a shadowing property of the method bound to the instance is made available. This pattern is similar to a common idiom used in JavaScript directly.
 
 ```mjs
 // bound.mjs
 export decorator @bound {
-  @register((target, name) => {
-    const method = target[name];
-    Object.defineProperty(target, name, {
-      get() {
-        const bound = method.bind(this);
-        Object.defineProperty(this, name, { value: bound, configurable: true });
-      }
-      configurable: true
-    });
-  })
+  @initialize(function(name) { this[name] = this[name].bind(this); })
```

Comment:
Is it possible to create a `@bound` decorator for private methods?

Reply:
Nope, not in this version. A later suite of decorators could provide that capability.